spark-cassandra-connector/src/it/scala/com/datastax/spark/connector/sql/CassandraDataFrameSpec.scala

```diff
@@ -81,13 +81,7 @@ class CassandraDataFrameSpec extends SparkCassandraITFlatSpecBase with Eventuall
     val df = sparkSession
       .read
       .format("org.apache.spark.sql.cassandra")
-      .options(
-        Map(
-          "table" -> "kv",
-          "keyspace" -> ks
-        )
-      )
-      .load()
+      .load(s"$ks.kv")
 
```

Comment:
I think we should have a test where we override the "options" , maybe there should be a message if you do that?